packages/module/patternfly-docs/content/extensions/chatbot/examples/UI/ChatbotHeaderDrawer.tsx

```diff
@@ -31,12 +31,29 @@ const initialConversations: { [key: string]: Conversation[] } = {
   ]
 };
 
+const ERROR = {
+  bodyText: (
+    <>
+      To try again, check your connection and refresh this page. If the issue persists,{' '}
```

Comment:
```suggestion
      To try again, check your connection and reload this page. If the issue persists,{' '}
```

Reply:
The button is cool with me! But actually can we tweak the text to just stick with the 'reload' terminology?

Reply:
All set!